apps/els_core/src/els_config.erl

```diff
@@ -24,6 +24,7 @@
 %% Macros
 %%==============================================================================
 -define(DEFAULT_CONFIG_FILE, "erlang_ls.config").
+-define(ALTERNATIVE_CONFIG_FILE, "erlang_ls.yml").
```

Comment:
@elbrujohalcon Is `.yml` or `.yaml` the most common extension?

Reply:
According to [The Great Oracle](https://stackoverflow.com/questions/21059124/is-it-yaml-or-yml), I should use `yaml`. Will change it in a moment.